Makefile

```diff
@@ -374,8 +338,21 @@ package-windows: require-version windows
 	@echo "Generating distribution package for windows/386..." && \
 	if [[ -z "$$SECRETS_DIR" ]]; then echo "SECRETS_DIR environment value is required."; exit 1; fi && \
 	if [[ -z "$$BNS_CERT_PASS" ]]; then echo "BNS_CERT_PASS environment value is required."; exit 1; fi && \
-	$(call docker-up) && \
```

Comment:
Note that by running this locally it's also a requisite to have `osslsigncode` and `makensis` locally as well, and we are not providing instructions for that anywhere, the person who is building needs to figure out how to do that instead of just using the docker image which already has those binaries.